### DISCUSSION TITLE
新的。。。不确定是使用不当还是BUG，但这是第一次看到DEEPL给我个报错出来！我认为报错很好，有助于找BUG。

### DISCUSSION THREAD

**Discussion by hmrg-grmh** [hcfyapp/crx-selection-translate#1057] [求助] [answered]
划词翻译版本：
![image](https://user-images.githubusercontent.com/68635334/119661600-07fd1b80-be63-11eb-93ee-0df122173ccc.png)

浏览器版本：
![image](https://user-images.githubusercontent.com/68635334/119661532-f9166900-be62-11eb-94a0-c084df2fa3d5.png)

操作系统：Win10

--------

我是在[这个地址](https://learnyousomeerlang.com/errors-and-processes)，先收回了之前的全文翻译，然后重新选了个词执行选词翻译，然后又摁了全文翻译，然后就有了（应该没记错）

![image](https://user-images.githubusercontent.com/68635334/119661335-c79d9d80-be62-11eb-820c-ca0153b58ac3.png)

这是新加的功能吗？DEEPL可以报错了，不过这个错没懂啥意思，我以前尝试全文和小窗同时搞也没事的啊。。。还是说多个自然段不能同时提交？但几十分钟前还可以的啊。。。

--------

![image](https://user-images.githubusercontent.com/68635334/119662434-f8ca9d80-be63-11eb-9b37-4b02a034a1e2.png)

重新选词也没再错。

另外，通应该是通的，网页能开也能 `ping` 通。

**Comment by lmk123** [comment, marked as answer]
这不是新加的功能，划词翻译里的 DeepL 一直都只支持同时处理一条翻译请求，只是在最近的版本中把这个报错抛出来了。

举例来说，你在网页上划词一次，会触发一次 DeepL 翻译；开启全文翻译，也会触发一次 DeepL 翻译。那么如果你组合起来：先划词一次，然后在这次翻译 DeepL 还没有获得结果的情况下，立刻开启全文翻译，此时由于你又触发了一次 DeepL 翻译而上一次的 DeepL 翻译还没处理完，那么上一次（也就是这个例子中划词操作触发的 DeepL 翻译）会被取消并显示这个错误。

全文翻译的时候，翻译请求是“串行”的而不是“并行”的，也就是等上一次可见区域内的文本翻译成功后，才会触发下一次的翻译，所以不会出现这个报错。

**Reply by hmrg-grmh** [reply]
ok 了解
